fusor-ember-cli/app/adapters/deployment-plan.js

```diff
@@ -1,5 +1,13 @@
 import DS from 'ember-data';
 
 export default DS.ActiveModelAdapter.extend({
-    namespace: 'fusor/api/openstack',
+    buildURL: function (type, query) {
```

Comment:
Do you call GET or PUT or DELETE on an individual `id` for `deployment-plan` `deployment-role`, `node`,  `flavor`, or `image`. If so, I need to modify the `buildURL` function